packages/flutter_tools/templates/list_detail_app/lib/src/settings/settings_service.dart.tmpl

```diff
@@ -0,0 +1,14 @@
+import 'package:flutter/material.dart';
+
+/// A service that stores and retrieves user settings.
+///
+/// By default, this class does not persist user settings. If you'd like to
+/// persist the user settings locally, use the shared_preferences package. If
+/// you'd like to store settings on a web server, use the http package.
+class SettingsService {
+  /// Loads the User's preferred ThemeMode from local or remote storage.
+  Future<ThemeMode> themeMode() async => ThemeMode.system;
+
+  /// Persists the user's preferred ThemeMode to local or remote storage.
+  Future<void> updateThemeMode(ThemeMode theme) async {}
```

Comment:
nit: maybe include a comment in the (currently empty) body of this method outlining would you could do here?

Reply:
(similar to what the class level comment already says, just to make it clear that this method is not supposed to be empty in a production app)